### PR TITLE
Add BoxyHQ Enterprise SSO

### PR DESCRIPTION
**Pull request by NathanTarbert**
Add BoxyHQ to Authentication & Identity

### CODE REVIEW

**Comment by agamm** [comment]
What are you competitors and how is BoxyHQ compare?

**Comment by agamm** [comment]
Ok. Please add a full stop at the end of the sentence. And is it possible you also add workos?

**Comment by NathanTarbert** [comment]
My bad for not adding the full stop at the end. I also added WorkOS and their tagline off of their website

**Review by fmerian — CHANGES_REQUESTED** [review]
hi @NathanTarbert, thanks for adding WorkOS, too.

can you make sure this new submission is sorted alphabetically please? 

thanks!

**Comment by NathanTarbert** [comment]
You’re welcome, no problem. Will sort alphabetically

**Comment by NathanTarbert** [comment]
Done!

File: README.md [hunk 1]
```diff
@@ -67,7 +67,8 @@ For the latest additions [click here](https://github.com/agamm/awesome-developer
 *Products that handle authentication and user identity for you.*
 * [Auth0](https://auth0.com/) - Authentication & authorization as a service.
 * [AWS cognito](https://aws.amazon.com/cognito/) - AWS's authentication as a service.
-* [BoxyHQ Enterprise SSO](https://boxyhq.com) - Open source API for enterprise SSO and Directory Sync
+* [BoxyHQ Enterprise SSO](https://boxyhq.com) - Open source API for enterprise SSO and Directory Sync.
+* [WorkOS](https://workos.com/) - Enterprise SSO (and a whole lot more).
```
Comment (agamm) [inline README.md]:
```suggestion
* [WorkOS](https://workos.com/) - Enterprise SSO and more out-of-the-box  enterprise features.
```

**Review by agamm — CHANGES_REQUESTED** [review]
Looks good, but sort it alphabetically with WorkOS?

**Comment by NathanTarbert** [comment]
Changed the description for WorkOS

**Review by agamm — APPROVED** [review]
Looks good!

**Review by fmerian — APPROVED** [review]

**Comment by NathanTarbert** [comment]
Hi, just curious when this PR will be merged? thanks

**Comment by NathanTarbert** [comment]
Thank you!

**Pull request merged**